js/ui/validator.js

```diff
@@ -219,66 +220,86 @@ var Validator = DOMComponent.inherit({
     * @publicName validate()
     * @return dxValidatorResult
     */
-    validate: function(args) {
-        var that = this,
-            adapter = that.option("adapter"),
-            name = that.option("name"),
+    validate(args) {
+        const adapter = this.option("adapter"),
+            name = this.option("name"),
             bypass = adapter.bypass && adapter.bypass(),
             value = (args && args.value !== undefined) ? args.value : adapter.getValue(),
             currentError = adapter.getCurrentValidationError && adapter.getCurrentValidationError(),
-            rules = this._getValidationRules(),
-            result;
-
+            rules = this._getValidationRules();
+        const currentResult = this._validationInfo && this._validationInfo.result;
+        if(currentResult && currentResult.status === "pending" && currentResult.value === value) {
+            return currentResult;
+        }
+        let result;
         if(bypass) {
             result = { isValid: true };
```

Comment:
Should it have the `status` field? 
Should we add specific method to get default or specific ValidationResult and use it in places like this or [L254](https://github.com/DevExpress/DevExtreme/pull/9655/files#diff-5f07271123f260b42dd2aaa84e126109L254)?

```js
validationEngine?
prepareValidationResult(result) {
    const defaultResult = {
        isValid: true,
        brokenRule: null,
        brokenRules: null,
        pendingRules: null,
        status: "valid",
        complete: null
    };
    return extend(true, defaultResult, result)
}

Reply:
>Should it have the status field?

Yes. This code for preventing validation when the status is pending and the value is not changed. In this case, the current validation result should be returned.

>Should we add specific method to get default or specific ValidationResult and use it in places like this or L254?

The result is reset only once. I do not think we should create a method for redefining the result object.